Resources/Locale/en-US/xenoarchaeology/traversal-distorter.ftl

```diff
@@ -0,0 +1,7 @@
+traversal-distorter-set-in = Traversal bias set to "in"
+traversal-distorter-set-out = Traversal bias set to "out"
+
+traversal-distorter-desc-in = The affected artifact's traversal now favors moving inwards to the beginning.
+traversal-distorter-desc-out = The affected artifact's traversal now favors moving outwards towards more dangerous nodes.
+
+traversal-distorter-upgrade-bias = Bias efficacy
```

Comment:
efficiency?

Reply:
Its the increase in how often the bias is applied. I guess "effectiveness" is probably a better word.